CMakeModules/CPackConfig.cmake

```diff
@@ -317,7 +313,7 @@ set(CPACK_RPM_PACKAGE_AUTOREQPROV " no")
 set(CPACK_RPM_PACKAGE_GROUP "Development/Libraries")
 set(CPACK_RPM_PACKAGE_LICENSE "BSD")
 set(CPACK_RPM_PACKAGE_URL "${SITE_URL}")
-if(AF_WITH_GRAPHICS)
+if(AF_BUILD_FORGE)
```

Comment:
These are optional now. You don't need these libs to run ArrayFire but they should be marked optional. use CPACK_RPM_PACKAGE_SUGGESTS instead of CPACK_RPM_PACKAGE_REQUIRES